### PR TITLE
[Fix] The localhost:8545 error in dashboard

### PR DESCRIPTION
**Pull request by kamikazebr**
Resolve https://github.com/1Hive/celeste-dashboard/issues/154

That PR make changes to check if  getDefaultProvider(null) is received, threat like getDefaultProvider(undefined) using the default values.

image

Too added jest files

image

## Another modifications

To be able to build that isseu the BundleWatch was disabled and 3box removed.

## How to test?

1 - Open https://celeste-dashboard-dk1sam5d7-1hive.vercel.app/
2 - Check Dev Tools Console

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
     "@aragon/connect": "^0.6.0",
     "@sentry/browser": "^5.12.1",
     "@web3-react/core": "^6.0.2",
     "@web3-react/injected-connector": "^6.0.3",
+    "3box": "=1.22.0",
     "clipboard-polyfill": "^2.8.6",
     "dayjs": "^1.8.14",
     "dotenv": "^8.2.0",
-    "ethers": "^5.0.9",
+    "ethers": "5.0.8",
```
[the remark this answers is not in the document]
Comment (kamikazebr) [reply package.json:17]: Nice question! To dedup the dependencies, we had both on that with that we get less bundle size in the build process.

Maybe we have another way to do that, what do you think?

[the remark this answers is not in the document]
Comment (kamikazebr) [reply package.json:17]: I can revert it too and open new issue to that.

**Comment by kamikazebr** [comment]
image
 Looks like that repo was archived, and its used for 3box lib.

**Comment by kamikazebr** [comment]
Curiously now the bundle its less 1MB. Maybe its possible even reative BundleWatch.

I'll let it to post merge that issue.

image

**Comment by 0xGabi** [comment]
Good call removing the deprecated package.  If we can activate BudleWatch again I think it make sense. 

About the dedup of ethers I would keep the latest version instead.

**Comment by kamikazebr** [comment]
image

**Comment by kamikazebr** [comment]
For some reason the build freeze, in localhost and at Vercel.

**Review by 0xGabi — APPROVED** [review]
🙌

**Pull request merged**